tests/e2e/tutorials/tutorialBase.js

```diff
@@ -346,9 +346,16 @@ class TutorialBase {
   }
 
   async takeScreenshot(screenshotTitle) {
+    // Generates an URL that points to the backend logs at this time
+    const snapshot_url = utils.getGrayLogSnapshotUrl(this.__url, 30);
```

Comment:
the __url variable contains the Domain and in the p2e case also part of the Path. You will need to extract the Hostname at some point

Reply:
thx. done with suggestion of @ignapas !